### DISCUSSION TITLE
Plugin de auto-complete de css custom properties variáveis para VSCode

### DISCUSSION THREAD

**Discussion by felquis** [frontendbr/forum#2374] [Perguntas e Respostas]
Gente, eu tenho muitas CSS Custom Properties nos projetos, mas o VSCode não faz auto-complete de todas elas, vocês usam algum plugin?

Encontrei na lata este https://marketplace.visualstudio.com/items?itemName=paradigm-sy.vscode-css-variables-autocomplete porém só tem 1 review, e 900 downloads, eu evito o máximo instalar plugins desnecessários.

Se você outra outro plugin, recomenda para mim?

Talvez este tópico seja relacionado ao 
https://github.com/frontendbr/forum/discussions/432

**Comment by felquis** [comment]
Testei algumas e apenas essa funcionou https://marketplace.visualstudio.com/items?itemName=vunguyentuan.vscode-css-variables

**Reply by romulosardinha** [reply]
Tenho usado este, CSS Variable Autocomplete. Me servindo bem por hora
